lib/parsers/index.js

```diff
@@ -8,6 +8,7 @@ function parse(subtitleText, inputExtension, options = {}) {
     case '.srt': return srt(subtitleText, options);
     case '.scc': return scc(subtitleText, options);
     case '.vtt': return vtt(subtitleText, options);
+    case '.dfxp':
```

Comment:
getExtension does not have the output '.dfxp'. If inputExtension is the output of getExtension this could never be the case. Are there other functions that are calling the parse() function?

Reply:
Never mind, I see that it is called from validators/index.js. It makes sense to pass ttml() function for all .dfxp requests. lgtm.